### PR TITLE
Catch up the rename of white-space-collapse to text-space-collapse

### PR DESCRIPTION
**Pull request by kojiishi**
The `white-space-collapse` was renamed to `text-space-collapse`,
with some values moved to `text-space-trim`, and moved to CSS
Text level 4.

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/25588)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/350667232)

File: css/css-text/white-space/white-space-collapsing-discard-001.xht [hunk 1]
```diff
@@ -4,14 +4,14 @@
...
-		<link rel="help" title="CSS Text Level 3: 4.1. White Space Collapsing: the ‘white-space-collapsing’ property" href="http://www.w3.org/TR/css-text-3/#white-space-collapsing" />
+		<link rel="help" href="http://www.w3.org/TR/css-text-4/#white-space-collapsing" />
 		<link rel="match" href="reference/white-space-collapsing-discard-001-ref.xht"/>
 		<meta name="flags" content="" />
 		<meta name="assert" content="The UA should discard all white space in the element when white-space-collapsing is set to discard." />
 		<style type="text/css">
 			<![CDATA[
 				.test span {
-					white-space-collapsing: discard;
+					text-space-collapsing: discard;
```
Comment (frivoal) [inline css/css-text/white-space/white-space-collapsing-discard-001.xht]: The property is now called `text-space-collapse` not `text-space-collapsing`.

File: css/css-text/white-space/white-space-collapsing-preserve-breaks-001.xht [hunk 2]
```diff
@@ -4,14 +4,14 @@
...
-		<link rel="help" title="CSS Text Level 3: 4.1. White Space Collapsing: the ‘white-space-collapsing’ property" href="http://www.w3.org/TR/css-text-3/#white-space-collapsing" />
+		<link rel="help" href="http://www.w3.org/TR/css-text-4/#white-space-collapsing" />
 		<link rel="match" href="reference/white-space-collapsing-preserve-breaks-001-ref.xht"/>
 		<meta name="flags" content="" />
 		<meta name="assert" content="The UA should collapse sequences of white space into a single character when white-space-collapsing is set to preserve." />
 		<style type="text/css">
 			<![CDATA[
 				.test span {
-					white-space-collapsing: preserve-breaks;
+					text-space-collapsing: preserve-breaks;
```
Comment (frivoal) [inline css/css-text/white-space/white-space-collapsing-preserve-breaks-001.xht]: The property is now called `text-space-collapse` not `text-space-collapsing`.

**Review by frivoal — CHANGES_REQUESTED** [review]

**Comment by frivoal** [comment]
Fixed in a separate PR (5ab74422c00). Closing.

**Pull request closed without merging**